From e813137dc700f960a8831df3bb70efb1b7f4b612 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 11 Jul 2018 13:18:29 +0300
Subject: [PATCH] board: habana: add dkms to the prerequisites list

dkms is one of the prerequisites - treat it as such.
Add dkms to the prerequisites list.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/local_install_packages b/board/habanalabs/common/local_install_packages
index 75118a4..3a0fad5 100755
--- a/board/habanalabs/common/local_install_packages
+++ b/board/habanalabs/common/local_install_packages
@@ -4,7 +4,7 @@
 dl_host_base="http://jenkins:8080/view/deployment/job"
 dl_suff="lastSuccessfulBuild/artifact/*zip*"	# the '*' are not for expansion
 dl_file="archive.zip"
-pre_pkg_lst="protobuf-compiler libprotoc-dev"
+pre_pkg_lst="protobuf-compiler libprotoc-dev dkms"
 habana_pkg_lst="habanalabs habanatools libhabana"
 
 function local_msg() {
@@ -37,7 +37,7 @@
 }
 
 function local_install_packages() {
-	local_msg "******** Installing habana python prerequisites ********"
+	local_msg "************ Installing habana prerequisites ***********"
 	sudo apt-get install ${pre_pkg_lst} -y	|| return 1;
 	local_msg "************ Installing new habana packages ************"
 	sudo apt update				|| return 1;
